packages/react-components/src/organisms/__tests__/SpeakerList.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { render } from '@testing-library/react';
+import { createEventResponse } from '@asap-hub/fixtures';
+import SpeakersList from '../SpeakersList';
+
+it('renders the speakers list', async () => {
+  const { getByText } = render(<SpeakersList {...createEventResponse()} />);
+
+  expect(getByText('The team one')).toBeVisible();
+  expect(getByText('To be announced')).toBeVisible();
+});
```

Comment:
I couldn't see tests for the 'user was not announced' functionality

Reply:
We seem to be quite light on test coverage. 

The fixtures should be simplified to contain only the basics. We can then extend to test further with expectations and test descriptions explaining what we are testing.